changelog/pending/20250325--sdk-python--add-getrequiredpackages-to-the-python-automation-api.yaml

```
@@ -0,0 +1,4 @@
+changes:
+- type: feat
+  scope: sdk/python
+  description: Add `GetRequiredPackages` to the Python Automation API
```

Since we're not really adding such a method to the auto API, but merely adding a stub implementation, maybe we can reword this:

```suggestion
  description: Avoid error messages due to missing a missing implementation of `GetRequiredPackages` in the Python Automation API server
```